ga4gh/ga2vcf.py

```diff
@@ -0,0 +1,345 @@
+"""
+Conversion utility that fetches data from the GA4GH server and
+writes to VCF files
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import re
+
+import pysam
+
+import ga4gh.ga2utils as ga2utils
+import ga4gh.request_factory as requestFactory
+
+
+class VcfException(Exception):
+    pass
+
+
+class VcfMetadataInfoLine(object):
+
```

Comment:
I'm a bit confused about what this class actually does. Why the `__getattr__` trick?